docs/automation.md

```diff
@@ -87,4 +87,59 @@ This is a snapshot of our automated checks at a glance.
 
 ![Continuous Deployment practices](./media/continuous_deployment_practices.png)
 
+## Lambda layer build & deploy
```

Comment:
I would like your review here @heitorlessa and @dreamorosi! I left a few lines commented in the text to see the before and after.

Reply:
Sure!!